src/components/Comments/Comment.less

```diff
@@ -50,9 +58,14 @@
   &__text {
     position: relative;
     top: -4px;
-    width: 100%;
+    width: calc(~'100% - 44px');
```

Comment:
it might be good to put this in `common.less` and accompany it with the avatar width:

```
@default-avatar-size: 40px;
@comment-text-offset: @default-avatar-size + 4px;
```

But see in `https://github.com/busyorg/busy/blob/new-design/src/components/Comments/Comment.js#L252` ... avatar size can be 40px or 32px, and I think default avatar in the `Avatar.js` component is 34px;

Reply:
Should I not better put in in `custom.less` instead of `common.less` ? I see that in `custom.less` are most of the variables. Please advise.

Reply:
yeah thats what i meant sorry, `custom.less`

Reply:
Thanks, I will update it shortly.